### ISSUE TITLE
Getting error units do not match

### ISSUE THREAD

**Issue by petermarks12** [josdejong/mathjs#1746]
I am just trying to create a unit degree/hour in frequency and i am using following logic.  Any solution to create such unit ?

`math.createUnit('degreehour', '1 deg/hour') `

I am getting this error:
Error: Units do not match ('hertz' != '1 degreehour')

**Comment by josdejong** [comment]
Thanks for reporting. How can I reproduce your issue? I don't get the error you report when doing:

```js
math.createUnit('degreehour', '1 deg/hour')
math.evaluate('5 degreehour * 2 hour').toString() // "10 deg"

**Comment by petermarks12** [comment]
Actually, I just figured out the problem is not while creating, but while doing the conversion.

Degree/hour is angular frequency and I am trying to convert it to hertz... at that point I am getting this error.  If you google it, convert deg/hour to hertz is possible, but here its not happening and I am getting this error.

**Comment by josdejong** [comment]
I think it is because `Hertz` has a different base unit (`FREQUENCY`) then `hour` has (`TIME`). @ericman314 is there a way to do such a conversion?

**Comment by ericman314** [comment]
Yes, you could create a unit for revolutions per second like this: `math.createUnit('rHz', '360 deg/s')`. Then the conversion from `degreehour` to `rHz` will work.

It's not possible to convert directly to `Hz` because one unit has angles and the other does not. It's impossible for math.js to know which conversion the user wants, because it might make sense to assume revolutions per second in one application, and radians per second in another. So that is why you have to define a new unit which includes the angle.

**Comment by petermarks12** [comment]
Also, I found a bug.

Year to Second should be 31536000, but I am getting 31557600.  Actually, Julian Year is equal to 31557600.

**Comment by josdejong** [comment]
mathjs has Julian year and month implemented (a year as 365.25 days). If you need different way of calculation I think you can define units for it yourself. And if you need a lot of precision I guess the best way is to calculate real durations between dates using a date library.
